Docker Installation (#16)
Human-Connection/Documentation: web-app/installation.md

# Docker Installation

If you have successfully installed the API, you have to install the web-app.

> Move to the parentdirectory of your API-Installation by typing

Reviewer: ```suggestion
Change to the parent directory from where you cloned the API repository:
```

```bash
   $ cd ..
   ```

> , clone the repo of Human Connection by typing

Reviewer: ```suggestion
, clone the WebApp repository of Human Connection by typing
```

 ```bash
   $ git clone https://github.com/Human-Connection/WebApp.git

   ```
and change to its directory with

```bash
   $ cd WebApp
   ```
.

> The following command, again, needs root-privileges, on Linux-Systems we strongly recommend the use of sudo, on Windows-Systems you will need admin-rights or you should use the LSS (Linux Sub System).

Reviewer: ```suggestion
Depending on your user's permissions, the following command, again, needs root-privileges. On Linux-Systems you can use `sudo`, on Windows-Systems you will need admin-rights or you can use the LSS (Linux Sub System).
```

> Determine the web-app of Human Connection by running 

Reviewer: ```suggestion
Setup the WebApp of Human Connection with one command:
```

```bash
$ sudo docker-compose up
```

> After this is done, reload your installation in your browser: [http://localhost:3030](http://localhost:3030), contributions at [http://localhost:3030/contributions](http://localhost:3030/contributions) or users at [http://localhost:3030/users](http://localhost:3030/users).

Reviewer: ```suggestion
```

> Congratulations! Now your web-app of Human Connection is running too. :-)

Reviewer: ```suggestion
This might take a while but after the installation has finished you can point your browser to [http://localhost:3000/](http://localhost:3000/) and see the login screen. Congratulations! :tada: 
```
